### DISCUSSION TITLE
Fixation button has been removed

### DISCUSSION THREAD

**Discussion by Luppyn** [webextensions/live-css-editor#91] [General]
I think in the last update the pin button was removed and it is a savior for me. I hope this was just a programming error and that get back to it soon, please.

**Comment by webextensions** [comment]
@Luppyn 
In the extension at the Chrome store, the last update was pushed around 2 months ago. Also, there have been no changes with regards to the pin button ("Apply styles automatically" feature).

Can you provide the following details:
* Browser version
* Extension version
* Operating System
* Does the issue happen on all sites or only a specific website / webpage?
* Screenshot of the extension on any webpage

**Reply by Luppyn** [reply]
- Firefox 105.0
- Magic CSS 8.10.2
- [AtlasOs](https://atlasos.net/download) 20H2 (probably)
- All sites
- ![image](https://user-images.githubusercontent.com/57575118/194439060-1c5ee2bd-4371-4d10-891f-1a2df75cce4e.png)

_**Remembering:** the button existed and worked perfectly._

**Reply by webextensions** [reply]
@Luppyn 
It appears that you are using an extension like https://addons.mozilla.org/en-US/firefox/addon/user-agent-string-switcher/ to switch the `navigator.userAgent`. The current code can misbehave in such a scenario.

For the time being, we would recommend you to use a userAgent which indicates the Firefox browser for the webpages where you wish to use Magic CSS.

An issue has been created at https://github.com/webextensions/live-css-editor/issues/92 for a proper fix.
